perftracker/templates/comparison_id.html

```diff
@@ -27,13 +27,39 @@ <h4>Job: {{ job }}</h4>
     </div>
   </div>
 </div>
-<span style="float: right">
-  <a onclick="show_all_tables()">Show tables</a>
-  <a onclick="hide_all_tables()">Hide tables</a>
-</span>
 
 <div id="main_content_container"></div>
 
+<button class="panel__btn_toggle"><i class="fa fa-angle-up"></i></button>
+<nav class="navbar-default navbar-fixed-bottom panel">
+  <div class="panel__button-controller panel__unit_hide-tables">
+      <h4><button onclick="hide_all_tables()">Hide tables</button></h4>
+  </div>
+  <div class="panel__button-controller panel__unit_show-tables">
+      <h4><button onclick="show_all_tables()">Show tables</button></h4>
+  </div>
+  <div class="panel__button-controller panel__unit_unselect">
+      <h4><button onclick="unselectGraphs()">Unselect All</button></h4>
+  </div>
+  <div class="panel__button-controller panel__unit_show-selected">
+      <h4><button onclick="showSelectedGraphs()">Show Selected Only</button></h4>
+  </div>
+  <div class="panel__button-controller panel__unit_go-to">
+      <h4><button>Group...</button></h4>
+      <div class="panel_dropup-content">
+          {% for g_tag, g in cmp_view.groups.items %}
+            {% if g.group_obj.title|length %}
+              <a href="#g_{{ g.id }}">{{ g.group_obj.title }}</a>
+            {% else %}
```

Comment:
no need for if, I guess template engine has better ways for defaults.
Example:
{{ value|default:"nothing" }}